script.lua

```diff
@@ -7786,6 +7803,8 @@ function spawnMaskedReminder(color, obj, isMarker)
         name = color.."'s "..obj.getName().." Reminder"
         tags = {"Destroy", "Mask", "Reminder Token"}
         scale = Vector(0.9, 0.9, 0.9)
+        scriptSuffix = "function onNumberTyped(_, num) Global.call(\"setReminderLabel\", {obj = self, num = num}); self.script_state = tostring(num) end"
+        onLoadSuffix = "self.max_typed_number = 99; if(saved_data ~= \"\") then onNumberTyped(nil, tonumber(saved_data)) end"
```

Comment:
Yeah, that was exactly what I thought at first, and was confused when it wasn't working. Only started working when I changed it to 99.

Reply:
It seems that the use of the word "determines" rather than "is" there is important. 'Cause if you set it to 2, then it triggers after 1 digit, but still triggers if that digit is greater than 2.